recipe/meta.yaml

```
@@ -33,17 +35,14 @@ test:
 
 about:
   home: https://github.com/pyca/cryptography
-  license: BSD-3-Clause OR Apache-2.0
-  license_family: BSD
-  license_file:
-    - LICENSE
-    - LICENSE.BSD
-    - LICENSE.APACHE
+  license: Apache-2.0 OR BSD-3-Clause
+  license_family: Apache
+  license_file: LICENSE
```

```suggestion
  license_family: OTHER
  license_file:
    - LICENSE
    - LICENSE.BSD
    - LICENSE.APACHE
```
Set the license family to other since the user can choose which license they want to use. As for license files, we must bundle all three files.

---

I had all license files in the recipe at first, but I was getting errors that they could not be found, so I removed them and kept `LICENSE` since it referred to both the Apache and BSD licenses. However, I will try again because maybe I had the wrong syntax and I did not have the family as `OTHER`.